OKR.md

```diff
@@ -2,6 +2,34 @@
 
 We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2018 Q3
+
+**The Connectivity Magic story is figured out and solved**
+
+- `P1` - @diasdavid -     A js-ipfs Daemon will always find other IPFS Daemons (js & go) in the LAN (MulticastDNS)
+- `P0` - @diasdavid -     The DHT is enabled by default in js-ipfs
+- `P1` - @diasdavid -     Circuit Relay Interop tests are finalized and merged (depends on go-ipfs 0.4.16)
+- `P0` - ???        -     A js-ipfs node (Browser & Desktop) always finds files that are on the Gateways
+
+**Daemon is as reliable as it can be**
+
+- `P0` - @alanshaw -      A Daemon runs for 10 days without a crash 
+- `P2` - ??? -            One or more js-ipfs daemons are part of the Bootsrapper nodes 
```

Comment:
@hugomrdias are you down to take this KR?

Reply:
yes